docs/sources/developers/http_api/apis.md

```diff
@@ -42,11 +42,27 @@ Groups organize related functionality into logical collections. For example `das
 
 These APIs will also uses semantic versioning with three stability levels:
 
-| Level | Format     | Description                                                                 | Use Case                 |
-| ----- | ---------- | --------------------------------------------------------------------------- | ------------------------ |
-| Alpha | `v1alpha1` | Early development stage. Unstable, may contain bugs, and subject to removal | For testing new features |
-| Beta  | `v1beta1`  | More stable than alpha, but may still have some changes                     | For early production use |
-| GA    | `v1`       | Generally Available. Stable with backward compatibility guarantees          | For production use       |
+| Level | Format     | Description                                                                 | Use Case                 | Enabled By Default? |
+| ----- | ---------- | --------------------------------------------------------------------------- | ------------------------ | ------------------- |
+| Alpha | `v1alpha1` | Early development stage. Unstable, may contain bugs, and subject to removal | For testing new features | No                  |
+| Beta  | `v1beta1`  | More stable than alpha, but may still have some changes                     | For early production use | No                  |
```

Comment:
I don't think we should recommend using beta versions in production. To me those are better suited for something like "early preview" stage or something similar, WDYT?
```suggestion
| Beta  | `v1beta1`  | More stable than alpha, but may still have some changes                     | For non-critical use | No                  |
```